sbt-dumpschema/src/main/scala/org/virtuslab/akkasaferserializer/DumpSchema.scala

```diff
@@ -0,0 +1,50 @@
+package org.virtuslab.akkasaferserializer
+
+import sbt.{File => JFile, _}
+import sbt.Keys._
+import better.files._
+
+import scala.util.parsing.json.{JSON, JSONArray, JSONFormat, JSONObject}
+
+object DumpSchema {
+  import DumpSchemaPlugin.autoImport._
+
+  def apply(outputFile: File, inputDirectory: File): Unit = {
+    val jsons = for {
+      file <- inputDirectory.list.toSeq.sorted(File.Order.byName)
+      line <- file.lineIterator
+      json <- JSON.parseRaw(line)
+    } yield json
+
+    for {
+      writer <- outputFile.clear().newFileOutputStream().printWriter().autoClosed
+    } writer.print(format(JSONArray(jsons.toList)))
+  }
+
+  private def format(t: Any, i: Int = 0): String =
+    t match {
+      case o: JSONObject =>
+        o.obj
+          .map {
+            case (k, v) =>
+              "  " * (i + 1) + JSONFormat.defaultFormatter(k) + ": " + format(v, i + 1)
+          }
+          .mkString("{\n", ",\n", "\n" + "  " * i + "}")
```

Comment:
Hmmm this looks like sth that should be sorted out by a JSON library under the hood rather than by application code manually/explicitly 🤔